packages/flutter/lib/src/gestures/tap_and_drag.dart

```diff
@@ -68,14 +73,13 @@ typedef GestureTapDragDownCallback  = void Function(TapDragDownDetails details);
 class TapDragDownDetails with Diagnosticable {
   /// Creates details for a [GestureTapDragDownCallback].
   ///
-  /// The [globalPosition], [localPosition], [consecutiveTapCount], and
-  /// [keysPressedOnDown] arguments must be provided and must not be null.
+  /// The [globalPosition], [localPosition] and [consecutiveTapCount]
```

Comment:
need a comma before `and`

Reply:
That's one more thing I just learned about American English then :) https://en.wikipedia.org/wiki/Serial_comma